include/ca_loop.h

```diff
@@ -0,0 +1,111 @@
+/**
+ * @brief RL Inference API definition.
+ *
+ * @file ca_loop.h
+ * @author Rajan Chari et al
+ * @date 2018-07-18
+ */
+#pragma once
+#include "action_flags.h"
+#include "base_loop.h"
+#include "continuous_action_response.h"
+#include "err_constants.h"
+#include "factory_resolver.h"
+#include "future_compat.h"
+#include "multi_slot_response.h"
+#include "multi_slot_response_detailed.h"
+#include "sender.h"
+
+#include <functional>
+#include <memory>
+
+namespace reinforcement_learning
```

Comment:
should the new api be on the same namespace?

Reply:
I dont see a reason to move it, this namespace is more general than just the live model